addons/procurement/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,50 @@
+---Fields in module 'procurement'---
+procurement  / procurement.order        / message_follower_ids (many2many)      : not a function anymore
+procurement  / procurement.order        / message_follower_ids (many2many)      : relation is now 'mail.followers' ('res.partner')
+procurement  / procurement.order        / message_follower_ids (many2many)      : type is now 'one2many' ('many2many')
+# Concerns module Mail (has to be done in another migration script specifically for that module) ---> Nothing to do
+
+procurement  / procurement.order        / message_channel_ids (many2many)       : NEW relation: mail.channel
+# New field ---> Nothing to do
+
+procurement  / procurement.order        / message_needaction (boolean)          : NEW
+# New field ---> Nothing to do
+
+procurement  / procurement.order        / message_needaction_counter (integer)  : NEW
+# New field ---> Nothing to do
+
+procurement  / procurement.order        / message_partner_ids (many2many)       : NEW relation: res.partner
+# New field ---> Nothing to do
+
+procurement  / procurement.order        / message_summary (text)                : DEL
+# Nothing to do
+
+procurement  / procurement.order        / message_unread_counter (integer)      : NEW
+# New field ---> Nothing to do
+
+procurement  / procurement.order        / product_uos (many2one)                : DEL relation: product.uom
```

Comment:
Not.

product_uos and product_uos_qty dissapear:
https://github.com/OCA/OpenUpgrade/blob/9.0/addons/sale/sale.py#L557
and product_uom is setted with product_uom and product_uom_qty.


I'm right?